app/assets/javascripts/discourse/tests/integration/components/user-menu/menu-test.js

```diff
@@ -49,7 +70,164 @@ module("Integration | Component | user-menu", function (hooks) {
     assert.strictEqual(tabs.length, 1);
     const preferencesTab = tabs[0];
     assert.ok(preferencesTab.href.endsWith("/u/eviltrout/preferences"));
-    assert.strictEqual(preferencesTab.getAttribute("data-tab-number"), "1");
+    assert.strictEqual(preferencesTab.getAttribute("data-tab-number"), "4");
     assert.strictEqual(preferencesTab.getAttribute("tabindex"), "-1");
   });
+
+  test("likes tab is hidden if currentUser's like notifications frequency is 'never'", async function (assert) {
```

Comment:
```suggestion
  test("likes tab is hidden if current user's like notifications frequency is 'never'", async function (assert) {
```

`currentUser` is an internal implementation detail.